### PR TITLE
Updating the type of ProduceRecord

### PR DESCRIPTION
**Pull request by anushreeringne**
key-value type changed from object to string

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json [hunk 1]
```diff
@@ -450,11 +450,11 @@
       "required": [ "value" ],
       "properties": {
         "value": {
-          "type": "object",
+          "type": "string",
```
Comment (yungezz) [inline specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: hello, is this management plan or data plan? is this a breaking change?

**Comment by yungezz** [comment]
hi @anushreeringne is this change a bug fixing of Swagger file, or new change? If new change, it will be breaking change to end users. thanks.

File: specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json (hunk 1 above)
Comment (anushreeringne) [reply specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: It supports type: string and type:int  which is why we put it as object.
Currently no customer is using this right now, so we are ok with updating the documentation with this change.

Should I use a type:string/int  ?Is it possible. If not, I prefer it to be updated to string, even though its breaking change.

Comment (yungezz) [reply specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: HI @anushreeringne , SDKs are generated based on Swagger. If Swagger changed with breaking change in same version, SDK will contain breaking change also. That's the concern. I don't think type could be `string/int`. Suggest to do it in a new version. definition of breaking change: https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#breaking-changes

**Pull request closed without merging**